components/MenuBar/MenuBar.jsx

```diff
@@ -1,77 +1,62 @@
 import React, { PropTypes, Component } from 'react'
+import classNames from 'classNames'
 
 require('./MenuBar.scss')
 
-class MenuBar extends Component {
-  constructor(props) {
-    super(props)
-
-    this.state = { mobile: false }
-    this.handleResize = this.handleResize.bind(this)
-  }
-
+export default class MenuBar extends Component {
   componentWillMount() {
     this.handleResize()
-    window.addEventListener('resize', this.handleResize)
+    window.addEventListener('resize', this.handleResize.bind(this))
   }
 
   componentWillUnmount() {
-    window.removeEventListener('resize', this.handleResize)
+    window.removeEventListener('resize', this.handleResize.bind(this))
```

Comment:
We may only need the `.bind(this)` in `componentWillMount` since that method is always called first. Not 100% sure though